temps.py

```diff
@@ -24,3 +24,10 @@ def convert_from_fahrenheit(temps):
         results.append((temp-32)/1.8)
 
     return results
+
+def convert_to_fahrenheit(temps):
+    results = []
+    for temp in temps:
+        results.append(temp*1.8+32)
```

Comment:
Here is a bug

Reply:
So many bugs in this code... please fix immediately.